### PR TITLE
Adds the Funny finger. A new way to show players your appreciation

### PR DESCRIPTION
**Pull request by ToGWtF**
### Intent of your Pull Request

Adds a cute new way to tell players just how much you like them!

Just point it their way and show them how you really feel!

haha funny finger go wew

hahafunnyfinger

### CODE REVIEW

**Comment by Partheo** [comment]
You finally got the sprite

**Comment by ToGWtF** [comment]
I made it. Thats why its so shite

**Review by Partheo — CHANGES_REQUESTED** [review]
Use the version Dead.OS made

**Review by DoubleTapper — COMMENTED** [review]
how do you get t h e f u n n y f i n g e r

**Review by Partheo — APPROVED** [review]

File: yogstation/code/modules/projectiles/guns/funnyfinger.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	item_flags = DROPDEL
+	max_charges = 1
+	recharge_rate = 4
+	ammo_type = /obj/item/ammo_casing/magic/funnyfinger
+	fire_sound_volume = 0
+	suppressed = TRUE
+	suppressed_sound = null
+	attack_verb = list("insulted", "cursed at", "swore at")
+
+/obj/item/gun/magic/funnyfinger/shoot_with_empty_chamber(mob/living/user as mob|obj)
```
Comment (alexkar598) [inline yogstation/code/modules/projectiles/guns/funnyfinger.dm:16]: Your mob cannot be an obj
```suggestion
/obj/item/gun/magic/funnyfinger/shoot_with_empty_chamber(mob/living/user as mob)
```

File: yogstation/code/modules/projectiles/guns/funnyfinger.dm [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
+	suppressed = TRUE
+	damage = 0
+	nodamage = TRUE
+	speed = 4
+	nondirectional_sprite = TRUE
+
+/obj/item/projectile/magic/funnyfinger/on_hit(atom/target, blocked = FALSE)
+	var/mob/living/L = target
+	to_chat(L, "<span class='userdanger'>Someone wanted you to know just how much they appreciate what you're doing!</span>")
```
Comment (alexkar598) [inline yogstation/code/modules/projectiles/guns/funnyfinger.dm:36]:
```suggestion
/obj/item/projectile/magic/funnyfinger/on_hit(atom/target, blocked = FALSE)
	if(isliving(target))
		var/mob/living/L = target
		to_chat(L, "<span class='userdanger'>Someone wanted you to know just how much they appreciate what you're doing!</span>")
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by alexkar598** [comment]
This PR has been closed due to being stale. Contact me or another maintainer on Discord if you wish for this PR to be reopened.

**Pull request closed without merging**